src/app/modules/dialogs/weekly-profit-dialog/weekly-profit-dialog.component.ts

```diff
@@ -32,6 +108,76 @@ export class WeeklyProfitDialogComponent extends ChartGeneralMethodsComponent im
         ['Weekly Profit K$', 'right', '#0085ff'],
         ['All profit M$', '1', '#eeb000']
       ]);
+      this.loadSecondChart(data);
+
     });
   }
+
+  getPlatformFromVaultName(vaultName: string): string {
+      if (vaultName.startsWith('V_')) {
+          vaultName = vaultName.replace('V_', '');
+      }
+      const underscoreCount = vaultName.split('_').length;
```

Comment:
I had mistake when provided this method
use
      const underscoreCount = vaultName.split('_')?.length ?? 0;
      let platform = 'SINGLE';
      if (underscoreCount > 1) {